WORKSPACE

```diff
@@ -4,17 +4,19 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 git_repository(
     name = "build_bazel_rules_apple",
     remote = "https://github.com/bazelbuild/rules_apple.git",
-    # This is 1 commit behind 0.16.1 which upgraded to a broken verison of
-    # rules_swift on Bazel 0.24.1
-    commit = "eaf5817105d1aa8c473b4e5c3ddfdc34b36b071f",
+    tag = "0.18.0",
 )
 
 load(
     "@build_bazel_rules_apple//apple:repositories.bzl",
     "apple_rules_dependencies",
 )
 
-apple_rules_dependencies()
+git_repository(
+    name = "build_bazel_rules_swift",
+    remote = "https://github.com/bazelbuild/rules_swift.git",
+    commit = "0192f16b82b2998d846c45187545e38548a6671a",
```

Comment:
Is there not an official tagged release?

Reply:
Not yet, they just merged this which fixes the swift error we were hitting with `swift_c_module`